drivers/include/net/netdev2.h

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *               2015 Ell-i open source co-operative
+ *               2015 Freie Universität Berlin
+ *               2014 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @defgroup    net_netdev2 Low-Level Network Driver Interface
+ * @ingroup     net_netdev2
```

Comment:
Right. Sorry about that. But then the group should also be called `drivers_netdev2`.